addons/l10n_in/models/account_invoice.py

```diff
@@ -22,7 +27,7 @@ def _compute_amount_total_words(self):
             ('overseas', 'Overseas'),
             ('special_economic_zone', 'Special Economic Zone'),
             ('deemed_export', 'Deemed Export')
-        ], string="GST Treatment", readonly=True, states={'draft': [('readonly', False)]})
+        ], string="GST Treatment", compute="_compute_l10n_in_gst_treatment", store=True, readonly=True, states={'draft': [('readonly', False)]})
```

Comment:
This indend. This line is far too long.
Also, this should be `readonly=False`.
`states={'draft': [('readonly', False)]}` doesn't work with computed editable fields

Reply:
`_onchange_partner_id` override can be removed to be consistent